src/angular-app/languageforge/lexicon/directive/dc-semanticdomain.js

```diff
@@ -12,9 +12,8 @@ angular.module('palaso.ui.dc.semanticdomain', [])
       model: '=',
       selectField: '&'
     },
-    controller: ['$scope', '$state', 'lexRightsService', function ($scope, $state, rights) {
+    controller: ['$scope', '$state', 'lexRightsService', function ($scope, $state, rightsService) {
```

Comment:
Is this confined to being a LexiconRightsService?  If not then there's an opportunity to use it elsewhere, there's rights use above that uses the session service for rights.  No problem with this code, just pointing out the possible opportunity for consolidating.